org.gimp.GIMP.json

```diff
@@ -2,7 +2,7 @@
     "app-id": "org.gimp.GIMP",
     "branch": "stable",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "40",
+    "runtime-version": "41",
```

Comment:
no. don 't. This require plugins to be updated too.

Reply:
Ah ok, thanks!

Reply:
I was only commenting on the runtime version here. I case that wasn't clear.

Reply:
yep I got that :) I saw that there's another update that's updating deps, that's why I closed this. Once the other will be accepted, I'll update the remaining bits!